src/app/modules/group/pages/current-user/current-user.component.ts

```diff
@@ -36,4 +37,19 @@ export class CurrentUserComponent {
     });
   }
 
+  onModify(): void {
+    window.open(
+      `${ environment.oauthServerUrl }?redirect_uri=${ location.origin }`,
+      'updateProfileWindow',
+      'popup,width=800,height=640'
+    );
+
+    const onProfileUpdated = (): void => {
+      this.userSessionService.refresh().subscribe();
```

Comment:
The refresh doesn't work for me when I close the popup window.
The behavior I have after saving in the popup is: the popup window displays our app in the popup.
What I expect when reading the issue: the popup closes itself and the user is refreshed on the tab that opened the popup.

Reply:
chrome and firefox

Reply:
I've changed close window approach, please try again

Reply:
OK I got it, it's because the feature has to be tested on localhost...
When using the branch, the save button redirects to the prod version of the app, not the branch version